tests/cache/tests.py

```diff
@@ -1155,6 +1155,20 @@ def test_incr_decr_timeout(self):
 
 class BaseMemcachedTests(BaseCacheTests):
 
+    def test_location_parsing(self):
+        locations = [
+            ['server1.tld', 'server2:11211'],
```

Comment:
You could add this test in a separate commit since the first two strings aren't related to the changes.